app/src/main/java/fr/free/nrw/commons/upload/UploadService.java

```diff
@@ -183,20 +186,35 @@ public int onStartCommand(Intent intent, int flags, int startId) {
 
     @SuppressLint("StringFormatInvalid")
     private void uploadContribution(Contribution contribution) {
-        InputStream fileInputStream;
-
+        InputStream fileInputStream=null;
+        InputStream tempFileInputStream=null;
+        ContentInfo contentInfo=null;
         String notificationTag = contribution.getLocalUri().toString();
 
         try {
-            //FIXME: Google Photos bug
             File file1 = new File(contribution.getLocalUri().getPath());
             fileInputStream = new FileInputStream(file1);
+            tempFileInputStream = new FileInputStream(file1);
+            if(contentInfoUtil==null){
```

Comment:
Please use consistent formatting. :) E.g. see the catch block below for our usual bracket formatting